### PR TITLE
feat: use (only) dedicated charge led on compatible Jumper handsets

### LINKED ISSUE #5508: Charging LED's do not work on T20V2
### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### What part of EdgeTX is the focus of this bug?

Transmitter firmware

### Current Behavior

The CS switch LED's do not light up to show battery level when charging.

### Expected Behavior

CS LED's should indicate battery level when charging.

### Steps To Reproduce

Plug T20V2 into USB to charge batteries.

### Version

Nightly (Please give date/commit below)

### Transmitter

Jumper T20 V2

### Operating System (OS)

_No response_

### OS Version

_No response_

### Anything else?

Broken in PR #5232, works correctly before that.

### PR DESCRIPTION
**Pull request by 3djc**
This removes the "somewhat flaky" function led charging indication (imprecise because vbat calibration cannot be used) and replace it with hardware based charging led indication on following radios (white led turns off when charge is over)

-T-PRO S
-T12 MAX (tested ok)
-T14 (tested ok)
-T15 (tested ok)
-~~T20 V2~~ (hardware is capable, but led cannot be seen when radio is closed, so removing it)

On older Jumper radio with customisable switches, fixes #5508 broken by https://github.com/EdgeTX/edgetx/pull/5232

### CODE REVIEW

**Comment by 3djc** [comment]
For T-Pro S and T20v2, I was not able to notice a charging led, but those could be my protos. Left questions with Jumper

**Comment by philmoz** [comment]
TProS has an issue with bootloader - long press RTN or select Exit shuts down radio instead of starting firmware.

**Comment by 3djc** [comment]
> TProS has an issue with bootloader - long press RTN or select Exit shuts down radio instead of starting firmware.

I hardly see how this could be related to this PR since TproS doesn't define FUNCTION_SWITCHES, therefore, there is no code change for TProS

**Comment by philmoz** [comment]
> > TProS has an issue with bootloader - long press RTN or select Exit shuts down radio instead of starting firmware.
> 
> I hardly see how this could be related to this PR since TproS doesn't define FUNCTION_SWITCHES, therefore, there is no code change for TProS

Current main branch works fine. This PR rebased on current main does not.

**Comment by pfeerick** [comment]
I'm seeing this with T14, T-Pro S, T12-Max also. Not T15. 

Either something changed in main since this was branched off, or this PR is doing something else still, as the latest nightly is fine.

**Comment by 3djc** [comment]
ok, BW issue, will fix it somehow

**Comment by pfeerick** [comment]
NOT T20v2

File: radio/src/targets/taranis/board.cpp [hunk 1]
```diff
@@ -72,6 +72,20 @@ void boardInit()
...
+  // prime debounce state...
+  usbPlugged();
+
+  if (usbPlugged()) {
+      delaysInit();
+      while (usbPlugged()) {
+         delay_ms(1000);
+      }
+  }
+  pwrOff();
```
Comment (pfeerick) [inline radio/src/targets/taranis/board.cpp:86]: Shouldn't pwrOff(); be inside the if?

**Comment by 3djc** [comment]
> **_NOT_** T20v2 ;)

it does have function switches, not the others

**Comment by pfeerick** [comment]
You miss the point... it doesn't have the aforementioned problem ;)

**Comment by 3djc** [comment]
> You miss the point... it doesn't have the aforementioned problem ;)

it doesn't get it because it does have function switches

**Comment by 3djc** [comment]
Your (obviously) both right !

**Review by pfeerick — APPROVED** [review]
T-Pro S booted fine after pwrOff() tweaked, T12max, T15, T15 were all good other than that gremlin so should be fine also.

**Pull request merged**